java/src/main/java/Properties.java

```diff
@@ -0,0 +1,50 @@
+import org.json.JSONObject;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Map;
+
+public class Properties {
```

Comment:
This is actually `ObjectConfig`. `properties` is just a **constructor** argument of this class.

`ObjectConfig` class' responsibilities:
- Accept a common input object, which could be a instance from `jsoup` package.
- Build an object with the `properties` given in the PureHTML configuration.
  - For each property, it should execute sub configuration (`Config` instance) and assign its result to that property.
  - Example: `{ "a": subConfigA.extract(), "b": subConfigB.extract() }`